pydm/widgets/spinbox.py

```diff
@@ -24,6 +24,9 @@ def __init__(self, parent=None, init_channel=None):
         self.setDecimals(0)
         self.app = QApplication.instance()
 
+        self.valueChanged.connect(self.send_value)  # signal from spinbox
```

Comment:
I still need to test it but for saving some time I will check with you...
I believe that this particular change is going to revert the idea of only sending the value when "Return" is pressed. @mattgibbs and I were very concerned that the current behavior of UP + DOWN alone would lead to serious errors if one make a step that is not supposed to be made. I would like to keep the same `Return Pressed` behavior.
Would you mind to send me the exception that you are seeing? I believe that my suggestion down below will solve the locale problem...

Reply:
You are correct @hhslepicka. Sorry about that, I will return to the default treatment of events.
However there is an issue when we type a value in the lineEdit, do not press Return and the widget lose focus the value that is shown is the one we typed. I think the correct behavior should be to return to the pv value, correct?